ansible/rke2/rke2-playbook.yml

```diff
@@ -108,4 +110,27 @@
         CNI: "{{ cni }}"
         NODE_TOKEN: "{{ node_token }}"
       register: script_result
-      when: ansible_role == "worker"
\ No newline at end of file
+      when: ansible_role == "worker"
+
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  vars_files:
+    - vars.yaml
+  tasks:
+    - name: Check if all RKE2 pods are running
+      kubernetes.core.k8s_info:
+        kubeconfig: "{{ kubeconfig_file }}"
+        api_version: v1
+        kind: Pod
+        namespace: kube-system
+        label_selectors:
+          - app.kubernetes.io/managed-by=Helm
```

Comment:
I don't remember for sure, but I believe there are some system-level pods that don't have this label. That being said, if they are failing, then generally there is at least one pod with this label that will also be failing, so this should be sufficient. Just wanted to note in case we see some weirdness ever.

Reply:
Yes that should work! Good call. Created https://github.com/rancher/qa-tasks/issues/1842 and just put it in both PIT and RAFT projects for one of us to take.